src/item.cpp

```diff
@@ -751,7 +751,7 @@ void Item::serializeAttr(PropWriteStream& propWriteStream) const
 
 	if (hasAttribute(ITEM_ATTRIBUTE_DEFENSE)) {
 		propWriteStream.write<uint8_t>(ATTR_DEFENSE);
-		propWriteStream.write<int32_t>(getIntAttr(ITEM_ATTRIBUTE_DEFENSE));
+		propWriteStream.write<uint32_t>(getIntAttr(ITEM_ATTRIBUTE_DEFENSE));
```

Comment:
This is backwards-incompatible, do take care. It is better to wait until #2206 for items is ready.

Reply:
Oh okay... so should I put this PR on hold for now?

Reply:
You might keep working on it for science, but I would call it waste of time *for now*.